Fixes #408, #409, #410, #411, #412, #413, #417 (#416)
oauth-wg/oauth-sd-jwt-vc: draft-ietf-oauth-sd-jwt-vc.md

JWTs: For an SD-JWT document, a Holder can decide which claims to release (within
bounds defined by the Issuer).

> SD-JWT builds on JWT/JWS. It can also be used when there are no selectively

Reviewer: `"crit":["b64"]` and a detached payload is not supported, right?

https://datatracker.ietf.org/doc/html/rfc7797

Author: Answer was already provided by @bc-pi in https://github.com/oauth-wg/oauth-sd-jwt-vc/issues/417#issuecomment-4845240302. The intention here was to just fix a minor editorial issue.

   consistent with the intent of the provider of the Type Metadata.

> Type Metadata can be retrieved as described in (#retrieving-type-metadata).
> Ecosystems can define rules for a `vct` value that specify whether Type Metadata is to be processed and which Type Metadata retrieval methods Consumers are expected to support.

Reviewer: ```suggestion
Ecosystems can establish rules for the vct value to determine whether Type Metadata should be processed and which retrieval methods consumers must support.
```

Author: I don't think the proposed language changes anything in the proposed text. Of course, they can decide themselves whether or not they must/should/may process it and what retrieval methods must/should/may be supported. Additionally, it also removes a rather important reference to the corresponding section.
